src/osemgrep/cli_scan/Scan_CLI.ml

```diff
@@ -799,7 +803,8 @@ let cmdline_term ~allow_empty_config : conf Term.t =
           Engine_type.(PRO Language_only)
       | false, false, false, false -> default.engine_type
       | true, false, false, false when secrets ->
-          Error.abort "Mutually exclusive options --oss/--secrets"
+          Error.abort
+            "Mutually exclusive options --oss/--beta-testing-secrets-enabled"
```

Comment:
Same comment here on the message

Reply:
Actually, the only way to get here is if they passed the `--oss` flag. This code is directly handling converting flags.